learning/ml5/9.1-object-detection/3_webcam_persistence/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/p5.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/addons/p5.sound.min.js"></script>
```

Comment:
```suggestion
```

Small thing but I think I prefer to remove p5.sound if it's not used by the example